src/api/TokenIOBuilder.h

```diff
@@ -35,6 +35,9 @@
 
 /// Crypto key storage. By default, uses Secure Enclave.
 @property (readwrite) id<TKKeyStore> keyStore;
+
+/// If you are using your own key storage, Token crypto engine will ask for local authentication when you sign data by default. Disables this will skip the local authentication.
+@property (readwrite) BOOL useLocalAuthentication;
```

Comment:
local is a bad name I think. Do they even need to pick? they can already set the whole key store. so if they don't can we pick the right one automatically?

Reply:
In the last minutes of confusedly reading internets, I ?think? the "local" comes from Apple's name for this.

Reply:
"Local Authentication" is a phrase in Apple document for popup the touch id or passcode UI. We'd better keep it consistent to enhance the readability.

Reply:
We need to pick because we don't want local authentication happen during our bamboo tests. This option can help us avoid the local authentication in test cases. which is necessary.

Reply:
Don't we control that by setting the keystore? Why do we need additional setting besides picking the keystore? The way I see it is there are 3 key stores:
- Secure Enclave
- "Local"
- Test

The "Local" and Test could be the same keystore with additional "ask for PIN code" flag.